quipucords/tests/report_utils.py

```diff
@@ -0,0 +1,45 @@
+"""Utility functions to help tests for API reports."""
+import json
+import tarfile
+from io import BytesIO
+from pathlib import Path
+
+from requests import Response
+
+
+def extract_tarball_from_response(
```

Comment:
When I wrote this, I did not know that `api.common_report.extract_tar_gz` existed, and it performs a very similar function. I think I should delete this new function and rewrite my tests to use `extract_tar_gz` instead.

Thoughts?

Reply:
such DRY

Reply:
in a serious note, yeah, I guess we should use that one.

OTOH, I just searched where in the codebase that function is used and it seems it is only being tested, but not used anywhere lol.

So... maybe just axe the other one and its tests?

Reply:
You know what, I just reread the referred function and yours and my verdict is: yours is better. The other is unused. IMO, keep this one.

Reply:
I'll take a few minutes to evaluate the other tests that use `extract_tar_gz`, try to make my new function work for both, and delete `extract_tar_gz`. 👍

Reply:
we can ignore the other one for now... we delete it in another PR (I still didn't found a good tool to detect unused code)

Reply:
Too late! 😂 I just pushed a couple commits that 1) refactor my function slightly, deletes extract_tar_gz, and update extract_tar_gz's callers, and 2) rewrites the offending unittest test class as pytest functions… because I touched it. 😛

Reply:
I'd resolve this thread, but I'll leave it to you since you opened it.